admin.php

```diff
@@ -104,14 +104,14 @@ function html() {
         echo '</table>';
 
         echo '<div class="pagenav">';
-        if($max < time()-(7*24*60*60)){
+        if ($max <= $_SERVER['REQUEST_TIME']){
```

Comment:
you should not rely on this HTTP related header. I guess replacing it by time() should work just as well.